addons/website/static/src/components/dialog/seo.js

```diff
@@ -483,6 +872,45 @@ export class OptimizeSEODialog extends Component {
                 'website_id': this.website.currentWebsite.id,
             },
         });
-        this.website.goToWebsite({path: this.url.replace(this.previousSeoName || this.seoNameDefault, seoContext.seoName)});
+
+        const rpcCalls = [];
+        if (
+            seoContext.brokenLinks.some(
+                (link) => link.oldLink !== link.newLink || link.remove === true
+            )
+        ) {
+            rpcCalls.push(
+                rpc("/website/update_broken_links", {
+                    res_id: this.object.id,
+                    res_model: this.object.model,
+                    links: seoContext.brokenLinks,
+                })
+            );
+            const footerEl = this.pageDocumentElement.querySelector("#footer");
+            if (footerEl) {
```

Comment:
Shouldn't we check here that there are `brokenLinks` in the footer before sending the `rpc` to avoid the unnecessary request?